include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -336,30 +336,31 @@ __parallel_transform_reduce(_ExecutionPolicy&& __exec, _Up __u, _Cp __combine, _
     __wgroup_size = ::std::min(__wgroup_size, oneapi::dpl::__internal::__kernel_work_group_size(
                                                   ::std::forward<_ExecutionPolicy>(__exec), __kernel));
 #endif
-    auto __mcu = oneapi::dpl::__internal::__max_compute_units(::std::forward<_ExecutionPolicy>(__exec));
-    auto __n_groups = (__n - 1) / __wgroup_size + 1;
-    __n_groups = ::std::min(decltype(__n_groups)(__mcu), __n_groups);
-    // TODO: try to change __n_groups with another formula for more perfect load balancing
+    // It can be replaced with another value for more perfect load balancing
+    auto __iters_per_witem = decltype(__wgroup_size)(4);
+    auto __size_per_wg = __iters_per_witem * __wgroup_size; // how much buffer elements work group processes
```

Comment:
`// how much buffer elements work group processes`

Hard to read. Do you mean ` // Number of buffer elements processed within workgroup`? Or do you mean something else?

Reply:
Your comment is also correct. I can change wording if it's better